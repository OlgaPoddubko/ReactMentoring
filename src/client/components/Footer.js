import React from 'react';

const Footer = props => (
  <div className="footer">
    <div className="inner-footer">netflixroulette</div>
    <style jsx>{`
      *{
        padding: 0;
        margin: 0;
        font-family: Candara, Arial, sans-serif;
      }

html, body {
  height: 100%;
}
#root{
  min-height: 100%;
  position: relative;
}
      .footer {
        position: absolute;
        bottom: 0;
        width: 100%;
        background: #414141;
      }

      .inner-footer {
        max-width: 1130px;
        margin: 0 auto;
        color: #e75f77;
        font-weight: bold;
        padding: 30px 35px;
      }
    `}</style>
  </div>
);

export default Footer;
