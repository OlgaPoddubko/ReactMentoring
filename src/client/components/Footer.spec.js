import React from 'react';
import Footer from './Footer';

describe('render Footer', () => {
  it('should render Footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
