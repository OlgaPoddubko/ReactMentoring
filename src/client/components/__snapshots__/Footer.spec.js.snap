// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Footer should render Footer 1`] = `
<div
  className="footer"
>
  <div
    className="inner-footer"
  >
    netflixroulette
  </div>
  <style
    jsx={true}
  >
    
      html, body {
        height: 100%;
      }

      #root{
        min-height: 100%;
        position: relative;
      }

      .footer {
        position: absolute;
        bottom: 0;
        width: 100%;
        background: #414141;
      }

      .inner-footer {
        max-width: 1130px;
        margin: 0 auto;
        color: #e75f77;
        font-weight: bold;
        padding: 30px 35px;
      }
    
  </style>
</div>
`;
